src/Libraries/CoreNodeModels/Input/BaseTypes.cs

```diff
@@ -123,7 +123,22 @@ public override string NodeType
         {
             get
             {
-                return "FloatInputNode";
+                return "NumberInputNode";
+            }
+        }
+
+        public string NumberType
+        {
+            get
+            {
+                if(this.GetType().ToString() == "CoreNodeModels.Input.DoubleInput")
+                {
+                    return "Double";
+                }
+                else
+                {
+                    return this.GetType().ToString();
+                }
```

Comment:
@mjkkirschner Not sure what we should do in the scenario that the JSON is modified and `NumberType` doesn't correspond with the `NodeType` as expected.  Do we throw an error, return the full type, or use a default value?